src/utils/lenkeUtil.tsx

```diff
@@ -30,6 +31,8 @@ export const lenkeTilModia = (personData: PersonData) => {
     path = `${path}/aktivitetskrav`;
   } else if (isGoingToBehandlerdialog) {
     path = `${path}/behandlerdialog`;
+  } else if (isGoingToSykmeldinger) {
+    path = `${path}/sykmeldinger`;
```

Comment:
Kanskje akkurat denne skulle hatt enda høyere "prioritet"?

Reply:
Hmja, kanskje. Noen tanker? Evt kan vi lage en lapp på å gå gjennom prioriteringene der.

Reply:
Jeg bare tenkte at det er viktigere at veilederen får med seg dette enn noe annet: feks hvis de trykker på aktivitetskravet, og vurderer dette før de ser at det er en oppgave på sykmeldingen, så kunne det vært en quirky feature at de kom rett inn på sykmeldingen når de trykker på personen fra oversikten, selv med akt.krav-filteret aktivt. Ville plassert denne øverst i så fall! 

Evt er det en egen lapp og en diskusjon med TH/Stine om hva som er mest riktig 🤔

Reply:
Ja, gir mening det for så vidt. Stemmer uansett for en egen lapp på det så kan vi få denne ut klar til prod :cowboy_hat_face:

Reply:
https://trello.com/c/z6zDRI0c/1725-prioritering-p%C3%A5-lenke-til-modia-fra-syfooversikt